### PR TITLE
Actualise package

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️

### CODE REVIEW

File: src/Generator/AbstractGenerator.php [hunk 1]
```diff
@@ -172,15 +170,12 @@ public function getDescription(): string
...
+    final public function validate(): Result
     {
         /** @psalm-suppress PossiblyInvalidArgument */
-        $results = (new Validator())->validate($this, $this->rules());
-        foreach ($results as $attribute => $resultItem) {
-            if (!$resultItem->isValid()) {
-                $this->errors[$attribute] = $resultItem->getErrors();
-            }
-        }
+        $results = $this->validator->validate($this, $this->rules());
```
Comment (arogachev) [inline src/Generator/AbstractGenerator.php:176]:
```suggestion
        $result = $this->validator->validate($this, $this->rules());
```

File: src/Generator/AbstractGenerator.php [hunk 2]
```diff
@@ -172,15 +170,12 @@ public function getDescription(): string
...
+
+        $this->errors = $results->getErrorMessagesIndexedByAttribute();
         return $results;
```
Comment (arogachev) [inline src/Generator/AbstractGenerator.php:179]: Need to return errors.

**Review by arogachev — CHANGES_REQUESTED** [review]
:+1:

File: src/Generator/AbstractGenerator.php (hunk 2 above)
Comment (xepozz) [reply src/Generator/AbstractGenerator.php:179]: The result of the validation is enough.

File: src/Generator/Controller/default/controller.php [hunk 3]
```diff
@@ -7,27 +8,34 @@
 
 /* @var $generator Yiisoft\Yii\Gii\Generator\Controller\Generator */
 
+$classModifiers = [];
```
Comment (arogachev) [inline src/Generator/Controller/default/controller.php:11]:
```suggestion
$classDefinitionParts = [];
```

**Review by arogachev — APPROVED** [review]

**Pull request merged**